src/main/java/edu/utdallas/davisbase/representation/UpdateCommandRepresentation.java

```diff
@@ -0,0 +1,62 @@
+package edu.utdallas.davisbase.representation;
+
+import java.util.List;
+
+public class UpdateCommandRepresentation implements CommandRepresentation {
+
+  private String command;
+  private String table;
+  private List<String> columns;
+  private List<String> values;
+  private WhereExpression whereClause;
+
+  public UpdateCommandRepresentation(String command, String table, List<String> columns, List<String> values, WhereExpression whereClause) {
+    this.command = command;
+    this.table = table;
+    this.columns = columns;
+    this.values = values;
```

Comment:
```suggestion
    this.values = Collections.unmodifiableList(new ArrayList<>(values));
```

(compile-time defensive programming + run-time fail-fast immutability)